### PR TITLE
updated inlhpcconnectivity url and Linux instructions

### PR DESCRIPTION
**Pull request by dhill2522**
This is a simple update to the INL HPC Connectivity docs. It fixes a broken url and adds Linux instructions for the SSH Config.

### CODE REVIEW

**Review by permcody — DISMISSED** [review]

**Comment by permcody** [comment]
Something is wrong with the PR. We may need to rebase this before merging. It doesn't reference a ticket yet it is passing our pre-check. We can do this internally if you don't have time to fix it.

**Comment by dhill2522** [comment]
Do you want me to make a ticket and reference it then? I am not exactly sure what you are looking for.

**Comment by dhill2522** [comment]
I am also fine if you just handle it internally. Mostly I am just hoping future Linux devs don't run into the same issues I did.

**Comment by dhill2522** [comment]
It looks like the markdown to html renderer being used wanted another newline in the file, so the published docs don't render the second code block properly. Are you looking to handle this internally or should I update it? It is literally just one missing newline before line 27 as far as I can tell. Sorry to bother you about this.

**Comment by permcody** [comment]
Thanks, I have fixed both of these issues. Thanks for your contribution!

**Review by permcody — DISMISSED** [review]

**Comment by loganharbour** [comment]
Something's goofed up in formatting:
https://mooseframework.inl.gov/docs/PRs/14767/site/help/inlhpcconnectivity.html

File: modules/doc/content/help/inlhpcconnectivity.md [hunk 1]
```diff
@@ -21,6 +23,20 @@ Host moosebuild.inl.gov
  ProxyCommand nc -x localhost:5555 %h %p
 ```
 
+For Linux Systems use the following instead:
```
Comment (loganharbour) [inline modules/doc/content/help/inlhpcconnectivity.md]:
```suggestion
For Linux Systems use the following instead:

```

**Review by loganharbour — CHANGES_REQUESTED** [review]

**Review by loganharbour — APPROVED** [review]

**Review by permcody — APPROVED** [review]

**Comment by loganharbour** [comment]
@dhill2522 which distro were you using that required this change?

See: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/moose-users/k6N8PZzd2Pk/ISPFpfJnAAAJ

**Comment by dhill2522** [comment]
Fedora 31

That is very interesting. I guess I assumed wrongly that my config would generally work for Linux systems.

**Pull request merged**